src/sage/combinat/k_regular_sequence.py

```diff
@@ -6,21 +6,6 @@
 found, for example, on the :wikipedia:`k-regular_sequence` or in
```

Comment:
The `k` in  lines 3 and 5 could be removed.

Reply:
Not convinced; see below (https://github.com/sagemath/sage/pull/36011#issuecomment-1659826240)